### PR TITLE
GSoC post 3

### PR DESCRIPTION
**Pull request by isabellarussell**
Hi all, post is in final draft mode.

Needs the demo video - have indicated where to put it. But it's an mp4 file so am unsure how to code it :/ 
Will send to you @gdelavald and @ggazzo.

All: please add any content you feel is missing. @Hudell I know this is the first time you see it, so feel free to put your spin on it. 

It can be merged on Monday (I'm away this afternoon) so don't merge until I've seen it for the last time.

Cheers!

### CODE REVIEW

**Comment by geekgonecrazy** [comment]
Can probably use basic video element: https://www.w3schools.com/htmL/html5_video.asp

File: _posts/2018/08/2018-08-01-GSoC-Post-3.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Speech to text is a technology that has gained popularity recently, with some providers like Google, Microsoft and IBM improving their engines and providing some open APIs to translate audio files into textual content. However, since this is pretty new, there are still some challenges when trying to work with different providers, like different encodings and file formats that are expected on different APIs.
+
+Our idea was to provide an easy way to connect different speech-to-text solutions to Rocket.Chat. Currently we have only implemented the Speech-To-Text feature using the Google Speech API, but it is easily replaceable with other engines such as IBM Watson or Bing Speech API; we will investigate these other solutions at a later date.
+
+We are carrying out a review to make sure everything works properly but hope to go live with this feature towards the end of September to accompany that release.
+
+### About the creator: Karan Bedi
+
+Karan Bedi is an undergrad student pursuing a major in Applied Mathematics at IIT Roorkee.
```
Comment (gdelavald) [inline _posts/2018/08/2018-08-01-GSoC-Post-3.md]: We can probably add that he has contributed to Rocket.Chat on his own before applying for the GSOC.

Comment (isabellarussell) [reply _posts/2018/08/2018-08-01-GSoC-Post-3.md]: 👍

Comment (isabellarussell) [reply _posts/2018/08/2018-08-01-GSoC-Post-3.md]: On the questionnaire he said it was his first experience contributing to an OS project though?

Comment (gdelavald) [reply _posts/2018/08/2018-08-01-GSoC-Post-3.md]: He might have worked on it as a setup for the GSOC, so it works either way.

**Comment by Sing-Li** [comment]
Karan just shipped me this GIF to replace the `.mp4`

s2t-demo

Now we're ready to merge !

**Comment by Sing-Li** [comment]
For future reference, please get input from our design team --- @thiagosanchz  --- on adding "video" elements in the blog sections.  Any styling concern - etc.

Increasingly I think we will have demand for "video with voice content" in blogs and within other components of the website.

**Comment by kb0304** [comment]
@isabellarussell 
There is some text overflow on mobile devices.
text-overflow

**Comment by isabellarussell** [comment]
@ggazzo can you help with the text overflow on mobile as mentioned by @kb0304 above? Cheers!

**Comment by ggazzo** [comment]
@isabellarussell check again :)

**Pull request merged**